modules/Saritasa.Upsource/Saritasa.Upsource.psm1

```diff
@@ -0,0 +1,367 @@
+
+
+[string]
+$UpsourceUrl = ''
+[pscredential]
+$UpsourceCredentials
+
+
+function Initialize-Upsource {
+
+    [CmdletBinding()]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [string] $UpsourceUrl,
+        [Parameter(Mandatory = $true)]
+        [pscredential] $Credential
+    )
+
+    $script:UpsourceUrl = $UpsourceUrl
+    $script:UpsourceCredentials = $Credential
+}
+
+function Convert-Time {
```

Comment:
**Convert** is not valid verb.

    Get-Verb

Name if ConvertTime, since it's a local function, not a public cmdlet.